code/api/src/modules/user/resolvers.js

```diff
@@ -28,19 +28,26 @@ export async function create(parentValue, { name, email, password }) {
 }
 
 // Update user
-export async function update(parentValue, { id, primaryStyle, secondaryStyle }) {
-  // if(auth.user && auth.user.role === params.user.roles.user) {
-    return await models.User.update(
+export async function update(parentValue, { id, name, email, password, role, primaryStyle, secondaryStyle }) {
+  const user = await models.User.findOne({ where: { id } })
+
+
+  if(user) {
+    return await user.update(
       {
         id,
+        name,
+        email,
+        password,
+        role,
         primaryStyle,
         secondaryStyle
       },
       { where: { id } }
     )
-  // } else {
-  //   throw new Error('Operation denied.')
-  // }
+  } else {
+    throw new Error('Please login first to update information.')
+  }
```

Comment:
Great fix!

Reply:
Glad we could isolate the issues in this code.  So glad that we know our code is working!